SteakBot.Core/Services/MemeService.cs

```diff
@@ -52,6 +65,32 @@ public bool EditCommand(string oldCommandName, MemeCommand newCommand)
             return true;
         }
 
+        public MemeResultType GetMemeType(string value)
+        {
+            var valueToProcess = value.ToLower();
+
+            var isImage = _imageFormats.Any(imageFormat => valueToProcess.Contains(imageFormat));
+            var isVideo = _videoFormats.Any(videoFormat => valueToProcess.Contains(videoFormat));
+
+            if (isImage == isVideo)
```

Comment:
If these are both `False` we'll get some false results. If both are `True` then it can be `if (isImage && isVodeo)`.

Reply:
Corrected checks and added a regex to detect urls.